style.css

```diff
@@ -784,3 +865,139 @@ span {
     display: none;
   }
 }
+
+.popup {
+  display: grid;
+  padding: 16px;
+  font-size: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.btn-popup-close {
+  position: absolute;
+  top: 0%;
+  right: 5%;
+  background: none;
+}
+
+.card-popup {
+  overflow: hidden;
+  box-sizing: border-box;
+  border-radius: 16px;
+}
+
+.card-popup > img {
+  width: 100%;
+  min-height: 220px;
+  object-fit: cover;
+}
+
+.popup-title {
+  font-family: Inter, sans-serif;
+  font-weight: 600;
+  font-size: 2em;
+  line-height: 44px;
+  color: #172b4d;
+  text-align: left;
+}
+
+.popup-tags {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: flex-start;
+}
+
+.tag-popup {
+  background: #ebf0ee;
+  color: #3a4a42;
+  padding: 0.5em 0.75em 0.5em;
+  font-family: Inter, sans-serif;
+  font-weight: 600;
+  font-size: 0.75em;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+  border-radius: 0.25em;
+}
+
+.pop-up-btn {
+  display: flex;
+  justify-content: space-between;
+}
+
+.pop-up-btn-info {
+  display: flex;
+  margin-top: 1.5em;
+  margin-bottom: 1.5em;
+  border: none;
+  width: fit-content;
+  align-self: center;
+  background: #227452;
```

Comment:
- [ ] Kindly make sure you use the right `background-colors` for the pop-up buttons as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190452654-f37c2d00-8605-47c0-9221-f0a8ee1b31e7.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190452623-068eb33a-6a45-4866-9009-17b46ff86025.png)

Reply:
ok I changed the button colors, changed the height of the photo, and changed the popup section design

Reply:
@AshakaE I don't understand what change do you want me to make i did all the changes that were requested please help me to understand

Reply:
@AshakaE  I changed the photos, button, and design what else is there to do

Reply:
@david63011 Great! 👍